tests/unit/db/account_tests.py

```diff
@@ -498,6 +522,19 @@ def test_requests_usage_data(self):
         finally:
             self.client.disconnect()
 
+    def test_set_year_without_month_for_requests_usage_data(self):
+        """
+        Test raising an exception when retrieving requests usage data with only
+        year parameter
+        """
+        try:
+            self.client.connect()
+            year = datetime.now().year
```

Comment:
Is there any reason not just to use a hard-coded value in these tests? I'd rather if we were able to do that as it removes variability in the tests.